ethclient/bor_ethclient.go

```diff
@@ -0,0 +1,30 @@
+package ethclient
+
+import (
+	"context"
+
+	ethereum "github.com/maticnetwork/bor"
+	"github.com/maticnetwork/bor/common"
+	"github.com/maticnetwork/bor/core/types"
+)
+
+// GetRootHash returns the merkle root of the block headers
+func (ec *Client) GetRootHash(ctx context.Context, startBlockNumber uint64, endBlockNumber uint64) (string, error) {
+	var rootHash string
+	if err := ec.c.CallContext(ctx, &rootHash, "eth_getRootHash", startBlockNumber, endBlockNumber); err != nil {
```

Comment:
Is it that `startBlockNumber` <= `endBlockNumber` ? If yes, we can put a check, if it's not handled in called function.

Reply:
I will check and fix.

Reply:
Check is there at https://github.com/maticnetwork/bor/blob/develop/consensus/bor/api.go#L156-L158